avocado/utils/process.py

```diff
@@ -98,6 +98,18 @@ def __str__(self):
             return "CmdError"
 
 
+def sudo_available():
```

Comment:
OK, no strict feeling about it, mine is slightly inaccurate, but yours is hideously long :smile: I'll use yours in the v2. (how would you feel about `can_sudo`? It's still inaccurate, but sounds good)